Assets/Scripts/TimeScaleController.cs

```diff
@@ -0,0 +1,6 @@
+using UnityEngine;
+
+public class TimeScaleController : MonoBehaviour

```

Comment:
```suggestion
public static class TimeScaleController
```
De functie is static maar niet de class?